.github/workflows/release.yml

```diff
@@ -28,8 +28,13 @@ jobs:
       - name: Install 💾
         run: npm ci
 
-      - name: Version ⬆️
-        run: npm version --new-version ${{ github.event.release.tag_name }} --no-git-tag-version
+      - name: Get package.json version 👀
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@master
+
+      - name: Check package.json version matches release tag ✅
+        run: echo "Package.json version does not match release version" && exit 1
```

Comment:
I'm not sure how I feel about this line. This only triggers after the release is "made" in GitHub. That means that if this fails, we need to remove the release and the tag and do it again.

Thoughts?

Reply:
The release is made on GitHub, but it shouldn't be published to NPM. I don't think it's a problem if we delete a GitHub release and re-do it after.

Maybe we could somehow revert the GitHub release automatically?